modelforge/train/training.py

```diff
@@ -183,15 +189,25 @@ def _predict_dipole_moment(
         self, model_predictions: Dict[str, torch.Tensor], batch: BatchData
     ) -> torch.Tensor:
 
-        per_atom_charge = model_predictions["per_atom_charge"]
+        per_atom_charge = model_predictions["per_atom_charge"]  # Shape: [num_atoms]
+        positions = batch.nnp_input.positions  # Shape: [num_atoms, 3]
+        per_atom_charge = per_atom_charge.unsqueeze(-1)  # Shape: [num_atoms, 1]
+        per_atom_dipole_contrib = per_atom_charge * positions  # Shape: [num_atoms, 3]
```

Comment:
I think it would be good to add a comment that dipole moment is calculated by weighting charge by the position of the charge; this assumes that the center of mass of the system is at the origin.

Reply:
added comments and docstrings